db/compacted_db_impl.cc

```diff
@@ -71,26 +73,27 @@ std::vector<Status> CompactedDBImpl::MultiGet(const ReadOptions& options,
       reader_list.push_back(f.fd.table_reader);
     }
   }
-  std::vector<Status> statuses(keys.size(), Status::NotFound());
```

Comment:
What the status must be passed to us with the right size? Why cannot we size it inside the function like we did before in here?